twitter_harvester.py

```diff
@@ -134,7 +134,7 @@ def user_timeline(self):
 
                 except HTTPError as e:
                     if e.response.status_code == 401:
-                        account = "user {} (User ID: {})".format(u"screen_name",
+                        account = "user {} (User ID: {})".format(screen_name,
```

Comment:
This should be `account = u"user {} (User ID: {})".format(screen_name,`.

Also, there are other places in twitter_harvester.py where the `u` is misplaced and must be fixed.